SPECS/libreswan.spec

```diff
@@ -143,18 +156,12 @@ FS=$(pwd)
 rm -rf %{buildroot}/usr/share/doc/libreswan
 
 install -d -m 0755 %{buildroot}%{_rundir}/pluto
-# used when setting --perpeerlog without --perpeerlogbase
-install -d -m 0700 %{buildroot}%{_localstatedir}/log/pluto/peer
 install -d %{buildroot}%{_sbindir}
 
 install -d %{buildroot}%{_sysconfdir}/sysctl.d
 install -m 0644 packaging/fedora/libreswan-sysctl.conf \
   %{buildroot}%{_sysconfdir}/sysctl.d/50-libreswan.conf
 
-install -d %{buildroot}%{_tmpfilesdir}
-install -m 0644 packaging/fedora/libreswan-tmpfiles.conf  \
-  %{buildroot}%{_tmpfilesdir}/libreswan.conf
-
 mkdir -p %{buildroot}%{_libdir}/fipscheck
```

Comment:
Not present in the upstream package. Do we need it?

Reply:
No, we discussed that together, I disabled the fips part as it doesn't make much sense as we apply patches, it actually does not bring anything. Removed.